test/engine/ConditionsSpec.scala

```diff
@@ -55,27 +55,20 @@ class ConditionsSpec extends UnitSpec with MockitoSugar with WithFakeApplication
 
     forAll(scenarios) { (scenario: String, enrolments: Set[String], expectedResult: Boolean) =>
       s"be true whether the user has any business enrolments - scenario: $scenario" in {
```

Comment:
i know this line hasn't changed in this work, but usually the word 'whether' comes with an alternative - eg. 'whether or not'